jobs/SDE/Assignments/SDE-FE-Assignment.md

```diff
@@ -55,17 +52,18 @@ provide option to go back to main page
 
 ### What we care about
 - Architecture:
-    - How clean and reasonable is the separation between modules?
+    - How clean and reasonable is the separation between codes?
+    - How good FE framework specific implementations?
```

Comment:
This is too generic, what is the definition of "good"? Candidate can't really act on this.

Reply:
Agreed. I will remove this from here, and under clarity, I will add the framework specific guidelines to be followed in.